lib/generators/manageiq/plugin/plugin_generator.rb

```diff
@@ -43,6 +43,7 @@ def create_plugin_files
       chmod "bin", 0755 & ~File.umask, :verbose => false
       empty_directory_with_keep_file "bundler.d"
       template "config/settings.yml"
+      template "config/secrets.defaults.yml"
```

Comment:
minor but alphabetically this should come before config/settings.yml

Reply:
:+1: done